resources/base/console-ui.yaml

```diff
@@ -296,6 +296,11 @@ metadata:
     app.kubernetes.io/instance: minio-operator
     app.kubernetes.io/name: operator
 spec:
+  strategy:
```

Comment:
I don't think we need this right now since we default to 1 replica

Reply:
removed